doc/specifications.md

```diff
@@ -0,0 +1,24 @@
+# Specifications
+
+These specifications are about an alternative application that make accessing
+trains schedules.
+
+## Must-haves
+
+These features are mandatory before the first release:
+
+- Work on Android and iOS
+- Get the shortest path from one station to an other
+- Specify date of my trip
+- CI/CD
+
```

Comment:
Is the design of the application taken into account?

Reply:
Yep, it's not the roadmap it's the specifications. So it's just a list of features.
What we could had tho is 
```
- Appealing design and good UX
```